waiter/bin/run-using-k8s.sh

```diff
@@ -8,6 +8,10 @@
 # Runs Waiter, configured to use the Kubernetes scheduler.
 
 export WAITER_PORT=${1:-9091}
+export GRAPHITE_SERVER_PORT=5555
+export JWKS_PORT=8040
+export JWKS_SERVER_URL="http://127.0.0.1:${JWKS_PORT}/keys"
+export WAITER_TEST_JWT_ACCESS_TOKEN_URL="http://127.0.0.1:${JWKS_PORT}/get-token?host={HOST}"
```

Comment:
Please move the test script (for k8s and minimesos) changes to separate PRs.

Reply:
Done